migration/migrating-openshift-3-to-4.adoc

```diff
@@ -0,0 +1,92 @@
+[id="migrating-openshift-3-to-4"]
+= Migrating {product-title} version 3 to 4
+include::modules/common-attributes.adoc[]
+:context: migrating-openshift-3-to-4
+
+toc::[]
+
+You can migrate application workloads from {product-title} 3.7 to a later version with the Cluster Application Migration (CAM) tool. The CAM enables you to control the migration and to minimize application downtime. The CAM provides a web console and an API, based on Kubernetes custom resources, for migrating stateful application workloads, at the granularity of a namespace, from a source to a target cluster.
+
+You can migrate data to any storage class that is available on the target cluster, for example, from Red Hat Gluster Storage or NFS storage to Red Hat Ceph Storage.
+
+Optionally, you can migrate the {product-title} 3.7 (and later) control plane settings to {product-title} 4.x with the Control Plane Migration Assistant (CPMA). See xref:migration-understanding-cpma_{context}[].
+
+.Prerequisites
+
+* You must have `cluster-admin` privileges.
+* You must have `podman` installed.
+* The source cluster(s) must be {product-title} 3.7 or later.
+* The target cluster must be {product-title} 3.9 or later.
+* The replication repository object storage must support the S3 API, for example, NooBaa, AWS S3, and generic S3 storage providers.
```

Comment:
Consider adding a blurb like below to point out a link for an example of leveraging the upstream NooBaa for a local S3 solution.  (Once OCS 4 is GA we will want to update this and refer to a procedure for using a supported downstream build, for time being recommendation is to proceed with unsupported upstream example).

"Examples of deploying a local S3 provider leveraging the upstream NooBaa project or using AWS S3 may be found at: https://github.com/fusor/mig-operator/blob/release-1.0/docs/usage/ObjectStorage.md"

Reply:
@jwmatthews Is it OK to have links to github? We also have them for the migration CRs. I thought they were problematic.

Reply:
In this case yes. Support can approve links to GitHub, so long as they are used for example and non critical items.

Reply:
Looks like it's OK if they're examples, so I'll add something like this.

Reply:
Thanks @sferich888!